openreview/conference/builder.py

```diff
@@ -939,15 +931,23 @@ def create_blind_submissions(self, hide_fields=[], number=None):
             blind_note = self.client.post_note(blind_note)
 
             if self.submission_stage.public and 'venue' not in blind_content:
-                blind_content['_bibtex'] = tools.generate_bibtex(
+                bibtex = tools.generate_bibtex(
                     note=note,
                     venue_fullname=self.name,
                     url_forum=blind_note.id,
                     year=str(self.get_year()))
 
-                blind_note.content = blind_content
-
-                blind_note = self.client.post_note(blind_note)
+                revision_note = self.client.post_note(openreview.Note(
```

Comment:
are you adding the revision to the original note or blind note?

Reply:
they should all be revisions of the original note but I can double check